pkg/pipeline/conntrack/hash.go

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
For the directory structure, should conntrack be directly under pipeline or under transform, like the other network transforms?

Reply:
I don't think of the connection tracking as a network transform, but as a new pipeline stage.
It will emit new types of records (connection related) in addition to the flow log records.